### PR TITLE
Add a catchall route and show a 404 message on it

### PR DESCRIPTION
**Pull request by bantic**
refs #213

cc @sandersonet

### CODE REVIEW

**Comment by bantic** [comment]
@sandersonet Is this what you had in mind for #213?

Any url that ember attempts and fails to route will display this not-found template.

Looks like this:
image

**Comment by sandersonet** [comment]
:+1: Perfect!

File: app/templates/shared/-not-found-error.hbs [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+    {{message}} Maybe the URL you visited is no longer valid?
+  {{else}}
+    Maybe you mistyped the address? If not, the page may have moved.
+  {{/if}}
+  </p>
+</div>
+<ul>
+  <li><a href="http://status.aptible.com">Aptible Status</a></li>
+  <li><a href="https://twitter.com/aptiblestatus">@aptiblestatus</a></li>
+  <li>{{#link-to-aptible app="status"}}Contact Support{{/link-to-aptible}}</li>
```
Comment (sandersonet) [inline app/templates/shared/-not-found-error.hbs]: Should this be `app="support"` ?

Comment (bantic) [inline app/templates/shared/-not-found-error.hbs]: thanks, good catch. I fixed this and added an assertion in `link-to-aptible` that should help catch future mistakes with the app name.

**Pull request merged**